LEAF_Request_Portal/form.php

```diff
@@ -3629,4 +3629,150 @@ public function permanentlyDeleteRecord($recordID) {
             
         return 1;
     }
+
+    /**
+     * Purpose: Send reminder emails to users depending on current step of record
+     * @param $recordID
+     * @param $days
+     * @throws SmartyException
+     */
+    function sendReminderEmail($recordID, $days) {
+
+        // Lookup approvers of current record so we can notify
+        $vars = array(':recordID' => $recordID);
+        $approvers = $this->db->prepared_query('SELECT *, users.userID AS approverID 
```

Comment:
Do we need all the values returned from this query?  Looks like the function just uses four of them.  Remember what I said about "select *".

Reply:
All "select *" queries were moved to strings, modified to only include the data needed, and string added to prepared_query function.